### PR TITLE
Clicking logout button fix

### LINKED ISSUE #120: [BUG] Clicking ->] Logout button in the (R) user menu should redirect to / or /login
With the Login working correctly, and the UI styled for Login and Register, 

The Logout function should:

- Kill the session, etc.
- and Redirect the user to / or /login

It's hard to tell if the logout function actually works as there is no visual clue that you're logged out.

Add a toast, popper, snackbar or alert  to tell the user that he is either logged in or logged out.  

Also the Login should welcome the user by name or user id.

### PR DESCRIPTION
**Pull request by ratishjain12**
fixes #120 

@gbowne1 I have fixed redirecting issue as well as added toast messages

please review and merge it!

### CODE REVIEW

**Comment by gbowne1** [comment]
@ratishjain12 

Can you please provide some screenshots of your changes in your PR description?

**Review by gbowne1 — APPROVED** [review]
- [x] checked out this change locally for testing  and review 

This change appears to work. The toasts are working as well.  

- am able to login and logout 
- visible login and logout status 

Approving for merge 

The only change I would make from this is to use Alert and Snackbar from @mui but this works fine for now.

I noticed the addition of react-router, which iirc is for React Native. Does react-router-dom not have the same functionality?

Approving for merge

File: src/server/package-lock.json [hunk 1]
```diff
@@ -16,18 +16,22 @@
                 "express": "^4.18.2",
                 "express-session": "^1.17.3",
                 "jsonwebtoken": "^9.0.1",
+<<<<<<< HEAD
```
Comment (LOGESH-B) [inline src/server/package-lock.json:19]: Hii @gbowne1 
Could you know why this package-lock.json is changing with developers...
I seen this in other projects aswell
Wheather this impact on the project?

**Review by LOGESH-B — APPROVED** [review]
Hii @ratishjain12 
Thanks for your Contribution🙌
It gives us the enhanced user experiance with toast

File: src/server/package-lock.json (hunk 1 above)
Comment (gbowne1) [reply src/server/package-lock.json:19]: Yeah, it just means the incoming changes need to be accepted on your end. Do you see a line where it says incoming changes?  It happens.  I don't know the full reason behind why but it should not have any impact.  

Thank you for your contribution to CodeBooker @ratishjain12 and for the opportunity to review and merge your PR.

**Comment by ratishjain12** [comment]
> Hii @ratishjain12 
> Thanks for your Contribution🙌
> It gives us the enhanced user experiance with toast

@LOGESH-B your welcome, looking forward to improve this project by making great contributions 😁

**Pull request merged**